RFC/RFC-9-hosted-preview-service.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: doc
+rfc: 9
+date: 2022-09-28
+title: Scene Preview Server
+authors:
+- pentreathm
+status: DRAFT
+---
+
+# Abstract
+
+Today users that don't own a land or are not ready to push their content to production need to use a Heroku application container to be able to preview and share a scene with different stakeholders. This solution requires some extra configuration steps and, on the other hand, it will stop being free according to the [latest Heroku news](https://blog.heroku.com/next-chapter). To ease the life of the content creators, a new scene preview server will be created, leveraging the same steps than when uploading content to the Catalyst network and enabling an easy way to validate and experience the content before it is finally uploaded to the Genesis City.  
```

Comment:
This section rather an introduction instead of an abstract, the abstract should reflect a tiny short summary of the whole document

Reply:
updated